src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+function BalanceBox() {
+  const { balancesDecorated } = useAccount();
+
+  const [balance, sats] = balancesDecorated.accountBalance.split(" ");
+  return (
+    <div
+      className={
+        "w-full rounded-[6px]  h-[88px] flex flex-col items-center justify-center bg-white hover:bg-gray-50 text-black dark:bg-surface-02dp dark:text-white dark:hover:bg-surface-16dp shadow transition duration-150 "
```

Comment:
Can you please remove setting the height `h-[88px]` inside the component, otherwise setting it from outside like this `<BalanceBox className="h-28 grow"></BalanceBox>` is not possible. The inside overwrites the outside.

Reply:
Currently in `<DefaultView>` the height for the buttons is implied by setting a height on the child icon element. You have to change this to setting the height to the button and then also set the height on the `<BalanceBox>` element.